common/lib/xmodule/xmodule/js/spec/capa/display_spec.coffee

```diff
@@ -802,3 +802,53 @@ describe 'Problem', ->
 
       # verify that codemirror textarea has correct `aria-describedby` attribute value
       expect($(CodeMirrorTextArea).attr('aria-describedby')).toEqual('cm-editor-exit-message-101 status_101')
+
+
+  describe 'show answer button', ->
+
+    beforeEach ->
+      @problem = new Problem($('.xblock-student_view'))
+
+    it 'should become enabled after a radiobutton is selected', ->
+      html = '''
```

Comment:
@Rabia23 Can we move this a fixture and load it? I see it has been used in the file 3-4 times.